test/example_test.dart

```diff
@@ -1,7 +1,29 @@
+import 'dart:io';
```

Comment:
if you want to play with testing, you might want to checkout [this](https://patrol.leancode.co/). Disclaimer, I work on it :)